actions/scrape/action.yml

```diff
@@ -74,7 +74,13 @@ runs:
         # Persist image tag for later steps (manifest)
         echo "DOCKER_IMAGE_TAG=${DOCKER_IMAGE_TAG}" >> "${GITHUB_ENV}"
 
-        bash "${{ github.action_path }}/scrape.sh" "${{ inputs.state }}" "${DOCKER_IMAGE_TAG}" "$(pwd)" "${{ github.workspace }}" || true
+        # Use CA-specific script for California (MySQL-based)
```

Comment:
here the code looks for ca to route to the ca specific scraper. if not, it uses the standard one.